jp2_online/static/vendors/Chart.js/docs/00-Getting-Started.md

```diff
@@ -0,0 +1,120 @@
+---
```

Comment:
is having all the documentation of this external project downloaded and being part of our repository truly needed?